Source/sync.cpp

```diff
@@ -232,7 +232,7 @@ bool IsTSyncMonsterValidate(const TSyncMonster &monsterSync)
 {
 	const int monsterId = monsterSync._mndx;
 
-	if (monsterId < 0 || monsterId >= MaxMonsters)
+	if (static_cast<size_t>(monsterId) >= MaxMonsters)

```

Comment:
`_mndx` is already unsinged (`uint8_t`), probably we should just change `monsterId` to `size_t`?

Reply:
Done